src/Development/IDE/Types/Location.hs

```diff
@@ -50,24 +50,7 @@ instance IsString NormalizedFilePath where
 
 toNormalizedFilePath :: FilePath -> NormalizedFilePath
 toNormalizedFilePath "" = NormalizedFilePath ""
```

Comment:
Is this special case still correct?

Reply:
Yes, otherwise this will be normalized to `"."`which is not what we want.

Reply:
I’ve added a comment to that effect.